packages/common/src/initiatives/HubInitiatives.ts

```diff
@@ -124,15 +128,34 @@ export function editorToInitiative(
   editor: IHubInitiativeEditor,
   portal: IPortal
 ): IHubInitiative {
-  // remove the ephemeral props we graft on for the editor
+  const _metric = editor._metric;
+
+  // 1. remove the ephemeral props we graft onto the editor
+  delete editor._groups;
+  delete editor._thumbnail;
+  delete editor.view?.featuredImage;
+  delete editor._metric;
   delete editor._groups;
-  // clone into a HubInitiative
-  const initiative = cloneObject(editor) as IHubInitiative;
-  // ensure there's an org url key
+
+  // 2. clone into a HubInitiative and ensure there's an orgUrlKey
+  let initiative = cloneObject(editor) as IHubInitiative;
   initiative.orgUrlKey = editor.orgUrlKey ? editor.orgUrlKey : portal.urlKey;
-  // copy the location extent up one level
+
+  // 3. copy the location extent up one level
   initiative.extent = editor.location?.extent;
-  // return with a cast
+
+  // 4. handle configured metric:
+  //   a. transform editor values into metric + displayConfig
+  //   b. set metric and displayConfig on initiative
+  if (_metric && Object.keys(_metric).length) {
+    const metricId = _metric.id || createId(camelize(`${_metric.cardTitle}_`));
```

Comment:
Oh, good catch! sorry I missed this. I was trying to move us away from `editorContext` because it didn't actually seem necessary since we can grab the `id` off the metric config. Updating to `_metric.metricId` _appears_ to fix the issue, but let me know if you see any other concerns with this.